docs/docs/get-started/write-your-first-package.md

```diff
@@ -175,7 +169,8 @@ UUID           Name       Ports                                                S
 b6fc024deefe   postgres   postgres: 5432/tcp -> postgresql://127.0.0.1:59299   RUNNING
 ```
 
-### Review: Run Postgres
+<details><summary>Review: Run Postgres</summary>
```

Comment:
I understand your intent and the style you chose here - but given how important this section is, is there a way to make the font bigger for the `Review: Run Postgres` title for the expandable text section? 

Not a blocker - just a suggestion.

Reply:
made them all bold! making a bigger font is possible but requires more css skills than i currently have so will stick with bold for now and make them bigger later